database/factories/EventFactory.ts

```diff
@@ -0,0 +1,16 @@
+import Event from '../../app/Models/Event'
+import Factory from '@ioc:Adonis/Lucid/Factory'
+
+export default Factory.define(Event, ({ faker }: any) => {
+  return {
+    title: faker.random.word(),
+    description: faker.random.word(),
+    cover: faker.random.numeric(),
+    pre_payment: (faker.random.numeric(5) + faker.random.numeric(3)).toString(),
+    box_office: (faker.random.numeric(5) + faker.random.numeric(3)).toString(),
+    venue: faker.random.numeric(),
```

Comment:
wie findest du es wenn wir faker.datatype.uuid() benutzen, für die ref?

Reply:
Ich mache es erstmal so wie es in den Models ist. Sonst fliegt alles auseinander. 
Das passen wir an wennnwir die Models / Schemas ändern